asammdf/blocks/mdf_v3.py

```diff
@@ -1090,9 +1090,7 @@ def add_trigger(self, group, timestamp, pre_time=0, post_time=0, comment=""):
 
             group.trigger = trigger
 
-    def append(
-        self, signals, acquisition_info="Python", common_timebase=False, units=None
-    ):
+    def append(self, signals, comment="Python", common_timebase=False, units=None):

```

Comment:
I would like to have the same method signature for the `MDF3` and `MDF4` classes

Reply:
Ok then comment is the only argument we can keep since MDF version 3 and 2 don't have acq_name and acq_source. We can have **kwargs for support of acq_name and acq_source:

```python
def append(self, signals, comment="Python", common_timebase=False, units=None, **kwargs):
```

or alternatively, acq_name and acq_source cannot be pass in append() and must be set using attributes setters.

Reply:
I think adding dummy arguments to the  mdf3 class would be more explicit

Reply:
Oh you're right, I didn't think about this possibility! I will implement that. Thanks

Reply:
It's done. I don't know why the diff shows mdf_v3.py as completely different; just select "ignore whitespaces" and it's back to normal.